src/Microsoft.Health.Fhir.Core/Extensions/StringExtensions.cs

```diff
@@ -6,13 +6,21 @@
 using System;
 using System.Security.Cryptography;
 using System.Text;
+using EnsureThat;
 
-namespace Microsoft.Health.CosmosDb.Features.Storage
+namespace Microsoft.Health.Fhir.Core.Extensions
 {
-    internal class Hasher
+    public static class StringExtensions
     {
-        public static string ComputeHash(string data)
+        /// <summary>
+        /// Computes SHA256 hash based of <paramref name="data"/>.
+        /// </summary>
+        /// <param name="data">The data to compute hash on.</param>
+        /// <returns>The computed hash string.</returns>
+        public static string ComputeHash(this string data)
         {
+            EnsureArg.IsNotNull(data, nameof(data));
```

Comment:
Is hashing whitespace valid?

Reply:
if one wishes to compute hash of whitespace 😊